drafts/prometheus-alertmanager.adoc

```diff
@@ -0,0 +1,250 @@
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/u/34190173
+author_github: https://github.com/jennifer-c
+seo-title: Get alerts from your Open Liberty apps in Slack using Prometheus Alertmanager - OpenLiberty.io
+seo-description: we'll set up Prometheus Alertmanager to fire alerts to a specified Slack channel to notify you when, for example, your app's heap usage is too high. We'll configure Prometheus with alerting rules to receive certain alerts from Open Liberty, then we'll configure Prometheus Alertmanager to pass those alerts to a Slack channel.
+blog_description: "we'll set up Prometheus Alertmanager to fire alerts to a specified Slack channel to notify you when, for example, your app's heap usage is too high. We'll configure Prometheus with alerting rules to receive certain alerts from Open Liberty, then we'll configure Prometheus Alertmanager to pass those alerts to a Slack channel."
+---
+= Get alerts from your Open Liberty apps in Slack using Prometheus Alertmanager
+Jennifer Cheng <https://github.com/jennifer-c>
+
+Every application needs a strong monitoring system to catch unexpected issues, whether it's an overloaded heap or a slow-responding servlet. link:https://openliberty.io/guides/microprofile-metrics.html[MicroProfile Metrics] provides the ability to expose metrics for your application. Used in conjunction with the open source monitoring system link:https://prometheus.io/[Prometheus] and the link:https://prometheus.io/docs/alerting/overview/[Prometheus Alertmanager], we can build a strong foundation for monitoring your system and reacting quickly to issues.
+
+In this post, we'll set up Prometheus Alertmanager to fire alerts to a specified Slack channel to notify you when, for example, your app's heap usage is too high. We'll configure Prometheus with alerting rules to receive certain alerts from Open Liberty, then we'll configure Prometheus Alertmanager to pass those alerts to a Slack channel.
+
+Already familiar with Alertmanager? Check out link:https://github.com/jennifer-c/openliberty-alertmanager[a sample configuration for Open Liberty]. The repository also contains the configuration used as an example in this blog post, if you get stuck along the way.
+
+== What you'll need
+
+Before we start, make sure your app is configured with an Open Liberty server with the `mpMetrics-2.0` feature enabled.
+
+Download and extract both link:https://prometheus.io/download/#prometheus[Prometheus and Prometheus Alertmanager] to your laptop. We'll assume you know the basics of using Prometheus already.
+
+You also need a Slack channel to send alerts to, as well as a link:https://api.slack.com/messaging/webhooks[Slack webhook] for that channel.
+
+== Configure Prometheus to receive certain alerts from Open Liberty
+
+JENNIFER: Add a brief introduction here to say what the reader will achieve/learn from this section (configure Prometheus to work with OL and then set up alerting rules in Prometheus to xxxx).
```

Comment:
ADD: In this section, you'll configure Prometheus to get metrics from Open Liberty. Then, you'll set up alerting rules using PromQL queries to give the Alertmanager some alerts to send.